addons/web/static/lib/ace/mode-less.js

```diff
@@ -0,0 +1,206 @@
+define("ace/mode/less", ["require", "exports", "module", "ace/lib/oop", "ace/mode/text", "ace/tokenizer", "ace/mode/less_highlight_rules", "ace/mode/matching_brace_outdent", "ace/mode/folding/cstyle"], function(e, t) {
```

Comment:
@Gorash I dug up this file from an old indian branch and "deminified" it and change the define calls so that it seems to work... where should I find a real good version of the file ? I search the ace library github but I did not find something which looks like the mode-xml.js file...